pkg/liqonet/iptables/iptables.go

```diff
@@ -653,9 +904,14 @@ func (h IPTHandler) insertLiqoRuleIfNotExists(chain string, rule IPTableRule) er
 }
 
 // Function to update specific rules in a given chain.
-func (h IPTHandler) updateSpecificRulesPerChain(chain string, existingRules []string, newRules []IPTableRule) error {
+func (h IPTHandler) updateSpecificRulesPerChain(chain string, existingRules []string, newRules []IPTableRule, keepExistingRules, prepend bool) error {
+	// Get iptables table
 	table := getTableFromChain(chain)
+
 	for _, existingRuleString := range existingRules {
+		if keepExistingRules {
+			break
+		}
```

Comment:
Because updateSpecificRulesPerChain is used (through EnsureClusterForwardRules) during the reconcile mechanism of TunnelEndpoint that, without keeping existing rules, overwrites rules created by EnsureRulesForClustersForwarding.